sdk/servicebus/service-bus/src/util/utils.ts

```diff
@@ -485,3 +486,89 @@ export type EntityStatus =
   | "Renaming"
   | "Restoring"
   | "Unknown";
+
+/**
+ * An executor for a Promise (or function that returns a Promise) that obeys both a timeout and an
```

Comment:
Since `actionFn` is always a function, we cannot really pass a promise directly right?

```suggestion
 * An executor for a function that returns a Promise that obeys both a timeout and an
```

Reply:
Yes, that's true! I actually had a plan to let it do both but that's not true anymore. Will remove.